client/me/account/main.jsx

```diff
@@ -50,6 +51,7 @@ import { isRequestingMissingSites } from 'state/selectors';
 import _user from 'lib/user';
 
 const user = _user();
+const colorSchemeKey = 'calypso_preferences.colorScheme';
```

Comment:
I would opt for the trade off of repetition in the places that need this string over the indirection of declaring the variable at this higher scope - mostly just because the file is quite large and general (as opposed to having a distinct focus on `colorSchemes` - in which case it might make a little more sense to scope it here).
Though I'm just adding this as food for thought - feel free to leave as is!

Reply:
Oh, I see now, the higher scope is b/c it's used in more than one function now.

I'm actually ok with this level of indirection here.

Reply:
Yes, it's used in several places and by storing it in one place we can easily replace it. Was considering importing it from a constants file but that probably only makes sense once we'd use the key across various files.